### ISSUE TITLE
Installer problems. Can't do gulp build in /semantic after installing

### ISSUE THREAD

**Issue by gczh** [Semantic-Org/Semantic-UI#3526]
**Configuration**
OS X El Capitan 10.11.1
Node v4.2.3
Npm v2.14.7
SemanticUI v2.1.7 (as per /node_modules/semantic-ui)

**Issue**
Followed the documentation accordingly using the express install option and did `cd semantic` after getting the following:

![image](https://cloud.githubusercontent.com/assets/1908934/12054019/f569bdb6-af59-11e5-89f4-858517d4037f.png)

Did a `gulp build` following the instructions from the installer's output. However what I got was the following:

```
SOHAPPY at Hey-MacBook-Air in ~/Documents/Projects/Sample/semantic
$ gulp build
[01:05:05] Local gulp not found in ~/Documents/Projects/Sample/semantic
[01:05:05] Try running: npm install gulp
```

After which I tried to install gulp using `npm install gulp --save-dev` and ran `gulp build` in the /semantic directory where the following error greeted me:

````
$ gulp build
module.js:339
    throw err;
    ^

Error: Cannot find module 'gulp-help'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/Hey/Documents/Projects/Sample/semantic/gulpfile.js:6:18)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
````

What works for me would be to run `gulp build` in node_modules/semantic-ui/ where I would get the following error:

```
/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/tasks/build/css.js:32
    tasksCompleted++;
    ^

ReferenceError: tasksCompleted is not defined
    at maybeCallback (/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/tasks/build/css.js:32:5)
    at finish (/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/run-sequence/index.js:62:5)
    at runNextSet (/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/run-sequence/index.js:90:5)
    at Gulp.onTaskEnd (/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/run-sequence/index.js:77:5)
    at emitOne (events.js:82:20)
    at Gulp.emit (events.js:169:7)
    at Gulp.Orchestrator._emitTaskDone (/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/gulp/node_modules/orchestrator/index.js:264:8)
    at /Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/gulp/node_modules/orchestrator/index.js:275:23
    at finish (/Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
    at /Users/Hey/Documents/Projects/Sample/node_modules/semantic-ui/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:52:4

```

Does anyone have similar issues and ideas on how to go about fixing this? This seems to be an issue with the npm installer.

**Comment by amad4biz** [comment]
According to @Jluck the npm version 2 has issue with semantic-ui 2.1.7. May be try using semantic-2.1.6 or  use npm version 3. it should work. I hope this help

**Comment by amad4biz** [comment]
As per @Jluck "Its because peer dependencies were accidentally removed
If you upgrade to npm3 it'll be fixed
alternatively downgrade to 2.1.6 or wait for 2.1.8
im still at my folks place for the holidays"

**Comment by cueedee** [comment]
Just ran into this exact same issue too using `npm@2.14.14`.

A possible work-around would be to `npm install semantic-ui@2.1.6 --save-dev` first, and then `npm update` second.

**Comment by jlukic** [comment]
#3511 for earlier discussion

**Comment by tcstory** [comment]
I met the same problem,I use npm 3.x and os x

**Comment by jlukic** [comment]
Fixed in 2.1.8, not sure how this could occur with NPM3

**Comment by cueedee** [comment]
Confirmed for npm2. Thanks!
